quickstarts/library-process/README.md

```diff
@@ -78,186 +57,114 @@ $ oc new-app example-app-secret -p SECRET_NAME=businesscentral-app-secret
-        * List of directories from which archives will be copied into the deployment folder=
-        * Timer service data store refresh interval (in milliseconds)=30000
-        * KIE Server Container Memory Limit=1Gi
-        * Disable KIE Server Management=true
-        * KIE Server Startup Strategy=LocalContainersStartupStrategy
-        * RH-SSO URL=
-        * RH-SSO Realm name=
-        * KIE Server RH-SSO Client name=
-        * KIE Server RH-SSO Client Secret=
-        * RH-SSO Realm Admin Username=
-        * RH-SSO Realm Admin Password=
-        * RH-SSO Disable SSL Certificate Validation=false
-        * RH-SSO Principal Attribute=preferred_username
-        * LDAP Endpoint=
-        * LDAP Bind DN=
-        * LDAP Bind Credentials=
-        * LDAP JAAS Security Domain=
-        * LDAP Base DN=
-        * LDAP Base Search filter=
-        * LDAP Search scope=
-        * LDAP Search time limit=
-        * LDAP DN attribute=
-        * LDAP Parse username=
-        * LDAP Username begin string=
-        * LDAP Username end string=
-        * LDAP Role attributeID=
-        * LDAP Roles Search DN=
-        * LDAP Role search filter=
-        * LDAP Role recursion=
-        * LDAP Default role=
-        * LDAP Role name attribute ID=
-        * LDAP Role DN contains roleNameAttributeID=
-        * LDAP Role Attribute ID is DN=
-        * LDAP Referral user attribute ID=
+...
 
 --> Creating resources ...
     serviceaccount "myapp-kieserver" created
-    rolebinding "myapp-kieserver-view" created
+    rolebinding.authorization.openshift.io "myapp-kieserver-edit" created
     service "myapp-kieserver" created
     service "myapp-kieserver-ping" created
     service "myapp-postgresql" created
-    route "myapp-kieserver" created
-    route "secure-myapp-kieserver" created
-    imagestream "myapp-kieserver" created
-    buildconfig "myapp-kieserver" created
-    deploymentconfig "myapp-kieserver" created
-    deploymentconfig "myapp-postgresql" created
+    route.route.openshift.io "insecure-myapp-kieserver" created
+    route.route.openshift.io "myapp-kieserver" created
+    imagestream.image.openshift.io "myapp-kieserver" created
+    buildconfig.build.openshift.io "myapp-kieserver" created
+    deploymentconfig.apps.openshift.io "myapp-kieserver" created
+    deploymentconfig.apps.openshift.io "myapp-postgresql" created
     persistentvolumeclaim "myapp-postgresql-claim" created
 --> Success
-    Access your application via route 'myapp-kieserver-rhpam-kieserver.mycloud.com'
-    Access your application via route 'secure-myapp-kieserver-rhpam-kieserver.mycloud.com'
+    Access your application via route 'insecure-myapp-kieserver-rhpam-kieserver.apps.spolti.cloud' 
+    Access your application via route 'myapp-kieserver-rhpam-kieserver.apps.spolti.cloud' 
     Build scheduled, use 'oc logs -f bc/myapp-kieserver' to track its progress.
     Run 'oc status' to view your app.
 ```
```

Comment:
Do we need to have it there? previous oc-cli outputs were removed.

Reply:
we can remove, it just pollutes the readme.